addons/website/static/src/components/dialog/page_properties.js

```diff
@@ -186,10 +189,13 @@ export class PagePropertiesDialog extends FormViewDialog {
 
     clonePage() {
         this.dialog.add(DuplicatePageDialog, {
-            pageId: this.resId,
-            onDuplicate: () => {
+            pageIds: [this.resId],
+            onDuplicate: (duplicates) => {
                 this.props.close();
                 this.props.onClose();
+                if (duplicates.length === 1) {
+                    this.website.goToWebsite({ path: duplicates[0], edition: true });
+                }
```

Comment:
```suggestion
                this.website.goToWebsite({ path: duplicates[0], edition: true });
```

This condition can only be true all the time, isn't it? We are in the page properties dialog (so only editing a single page) and the code provide `pageIds` as `[this.resId]` so only one record